odxtools/companydata.py

```diff
@@ -148,19 +149,12 @@ class TeamMember:
 
     @staticmethod
     def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) -> "TeamMember":
-        odx_id = OdxLinkId.from_et(et_element, doc_frags)
-        assert odx_id is not None
-        short_name = et_element.findtext("SHORT-NAME")
-        assert short_name is not None
+        odx_id = odxrequire(OdxLinkId.from_et(et_element, doc_frags))
+        short_name = odxrequire(et_element.findtext("SHORT-NAME"))
```

Comment:
odxrequire checks that element is not None, but an empty string is not None, but is still not a valid short_name

Reply:
yes, ensuring that all content strings of the XML are valid would be another (quite large) PR because this would affect a substantial part of the XML tags containing "primitive" objects. I tried not to commingle too much into one here. (I did not always succeed.)